src/main/java/com/minecolonies/blockout/views/ScrollingView.java

```diff
@@ -52,21 +39,32 @@ private void setup()
         scrollbar.putInside(this);
     }
 
+    @NotNull
+    protected ScrollingContainer createScrollingContainer()
+    {
+        return new ScrollingContainer(this);
+    }
+
+    /**
+     * Load from xml.
+     *
+     * @param params xml parameters.
+     */
+    public ScrollingView(final PaneParams params)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1213)